amxmodx/amxmodx.cpp

```diff
@@ -23,15 +23,30 @@
 extern CFlagManager FlagMan;
 ke::Vector<CAdminData *> DynamicAdmins;
 
+const char *g_sInaccessibleXVars[] =
+{
+	"MaxClients",
+	"MapName",
+	"PluginName",
```

Comment:
Why PluginName and not plugin_name? I would prefer to wrap this in macro.

`REGISTER_PLUGIN("blabla, blabalala", "bla")`, it is to noisy and long to declare plugin information like that

Reply:
In issue ( #712 ) already something similar was discussed. I still tend to think that it is more convenient when you know which argument you are changing. And in the macro, as you know, even the name of the argument cannot be specified.